CHANGELOG.md

```diff
@@ -53,6 +53,8 @@ This release depends on, and has been tested with, the following Mapbox dependen
 - Mapbox Java `v6.6.0` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v6.6.0))
 - Mapbox Android Core `v5.0.1`
 - Mapbox Android Telemetry `v8.1.2`
+- Introduced `RoutesSetCallback` parameter to `MapboxNavigation#setNavigationRoutes`, which is called after the routes passed to `MapboxNavigation#setNavigationRoutes` are processed or are failed to be processed [#5945](https://github.com/mapbox/mapbox-navigation-android/pull/5945).
+- `RoutesObserver#onRoutesChanged` will not be triggered if the navigator fails to process routes passed via `MapboxNavigation#setNavigationRoutes` [#5945](https://github.com/mapbox/mapbox-navigation-android/pull/5945).
```

Comment:
Move changelog to the appropriate section.

Reply:
And update it according to the guidelines: https://github.com/mapbox/navigation-sdks/blob/main/documentation/android-changelog-guidelines.md.

Reply:
Thanks. How do you usually keep track that changelog is not messed up after rebase?

Reply:
By reviewing the changes myself. We should probably improve our validate-changelog.py script to verify that the changelog is located in the proper section. 
By the way, your changelog contains wrong links and dots are in the wrong places.

Reply:
If I were a lawyer, I could argue about the dots... The guideline says:
> Begin each changelog entry with a capitalized past tense verb and end it with a period.

That's exactly as I did it. It's different in examples tho. Corrected.